cabal-install/Distribution/Client/CmdConfigure.hs

```diff
@@ -83,7 +86,14 @@ configureAction (configFlags, configExFlags, installFlags, haddockFlags)
     baseCtx <- establishProjectBaseContext verbosity cliConfig
 
     -- Write out the @cabal.project.local@ so it gets picked up by the
-    -- planning phase.
+    -- planning phase. If old config exists, then print the contents
+    -- before overwriting
+    exists <- doesFileExist "cabal.project.local"
+    when exists $ do
+        putStrLn "file: cabal.project.local already exists. Now overwriting it."
```

Comment:
To signify that `cabal.project.local` is a file. I can rewrite it as `"cabal.project.local" file already exists. Now overwriting it.`